packages/map-template/src/components/WebComponentWrappers/Keyboard/Keyboard.jsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useRef } from 'react';
+
+/**
+ * React wrapper around the custom element <mi-keyboard>.
+ *
+ * @param {object} props
+ * @param {object} props.searchInputElement 
```

Comment:
Write a few words about what this is.

Reply:
✅

Reply:
Should it be a HTMLInputElement instead of an object?

Reply:
Done ✅